### PR TITLE
#161/UI/AnotherButtonsColors/Refactor

### LINKED ISSUE #223: #161/UI/AnotherColors/Refactor

### PR DESCRIPTION
**Pull request by KatarzynaChmielecka**
closes #223

### CODE REVIEW

**Review by MonikaKrella — COMMENTED** [review]
will be approved after merge development

File: src/Organisms/BurgerMenu/BurgerMenu.js [hunk 1]
```diff
@@ -28,7 +28,11 @@ const BurgerMenu = ({ open, setOpen }) => {
         <h3>Menu</h3>
       </HeaderMenuStyled>
       <MenuStyled>
-        <Link to="user" className="link" onClick={() => setOpen(false)}>
+        <Link
+          to="/user/:userId"
```
Comment (MonikaKrella) [inline src/Organisms/BurgerMenu/BurgerMenu.js:32]: "/:userId" must be dynamic:
-url when we came from role page:
image
-url when we came from burger menu:
image

File: src/Organisms/NavElement/NavElement.js [hunk 2]
```diff
@@ -14,15 +14,21 @@ import { useState } from 'react';
...
-        <div className="burger-wrapper">
-          <MdMenu className="burger-icon" onClick={() => setOpen(true)} />
-        </div>
+        {login || register || contact ? (
+          <div className="burger-wrapper" />
+        ) : (
+          <div className="burger-wrapper">
+            <MdMenu className="burger-icon" onClick={() => setOpen(true)} />
+          </div>
+        )}
```
Comment (MonikaKrella) [inline src/Organisms/NavElement/NavElement.js:31]: it works nice!
add it also to /forgot and /in-progress

**Review by MonikaKrella — CHANGES_REQUESTED** [review]

File: src/Organisms/NavElement/NavElement.js (hunk 2 above)
Comment (KatarzynaChmielecka) [reply src/Organisms/NavElement/NavElement.js:31]: If yes- done ;)

Comment (MonikaKrella) [reply src/Organisms/NavElement/NavElement.js:33]: exactly that, great!

**Pull request merged**